### ISSUE TITLE
Support directly connecting with tokens

### ISSUE THREAD

**Issue by Fleker** [hildjj/node-feedly#13]
Working on a webhook interaction with Feedly, each user will bring their own tokens. I would be remiss to hardcode it on the server. It'd be great if, instead of using a file to process the tokens, I could pass them directly into the `Feedly` constructor.

**Comment by hildjj** [comment]
I like it.  Would you be willing to pass in an expiry date also?  I would throw an exception if you supplied any of (access_token, refresh_token, expires) without specifying all three.

**Comment by Fleker** [comment]
Yeah, I'd pass in all three. Right now I'm manually setting these fields after construction:

```
f.state = {
    access_token: getAccessToken(),
    refresh_token: getRefreshToken(),
    expires: new Date("2018-12-08T01:54:19.154Z")
}
```
